mmv1/products/privilegedaccessmanager/Entitlement.yaml

```
@@ -0,0 +1,265 @@
+          The workflow can consist of multiple serial steps where each step defines who can act as Approver in that step and how many of those users should approve before the workflow moves to the next step.
+          This can be used to create approval workflows such as
+          * Require an approval from any user in a group G.
+          * Require an approval from any k number of users from a Group G.
+          * Require an approval from any user in a group G and then from a user U. etc.
+          A single user might be part of `approvers` ACL for multiple steps in this workflow but they can only approve once and that approval will only be considered to satisfy the approval step at which it was granted.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: requireApproverJustification
+            description: |
+              Optional. Do the approvers need to provide a justification for their actions?
+          - !ruby/object:Api::Type::Array
+            name: steps
+            required: true
+            description: |
+              List of approval steps in this workflow. These steps would be followed in the specified order sequentially.  1 step is supported for now.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Array
+                  name: approvers
+                  required: true
+                  min_size: 1
+                  max_size: 1
+                  description: |
+                    The potential set of approvers in this step.This list should contain at most one entry.
+                  item_type: !ruby/object:Api::Type::NestedObject
+                    properties:
+                      - !ruby/object:Api::Type::Array
+                        name: principals
+                        required: true
+                        min_size: 1
+                        is_set: true
+                        item_type: Api::Type::String
+                        item_validation: !ruby/object:Provider::Terraform::Validation
+                          function: 'validateDeletedPrincipals'
+                        description: |
+                          Users who are being allowed for the operation. Each entry should be a valid v1 IAM Principal Identifier. Format for these is documented at: https://cloud.google.com/iam/docs/principal-identifiers#v1
+                - !ruby/object:Api::Type::Integer
+                  name: approvalsNeeded
+                  description: |
+                    Required. How many users from the above list need to approve.
+                    If there are not enough distinct users in the list above then the workflow
+                    will indefinitely block. Should always be greater than 0. 1 is the only
+                    supported value for now.
```

```suggestion
                    will indefinitely block. Should always be greater than 0. Currently 1 is the only
                    supported value.
```

also remove the default value as this field is required anyway.